azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/constants.py

```diff
@@ -0,0 +1,70 @@
+from os import environ
+from pathlib import Path
+from sys import platform
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+_LOG_PATH_LINUX = "/var/log/applicationinsights"
+_LOG_PATH_WINDOWS = "\\LogFiles\\ApplicationInsights"
+# TODO: Enabled when duplciate logging issue is solved
+# __EXPORTER_DIAGNOSTICS_ENABLED_ENV_VAR = (
+#     "AZURE_MONITOR_OPENTELEMETRY_DISTRO_ENABLE_EXPORTER_DIAGNOSTICS"
+# )
+
+
+def _get_log_path(status_log_path=False):
+    if platform.startswith("linux"):
+        return _LOG_PATH_LINUX
+    elif platform.startswith("win32"):
+        log_path = str(Path.home()) + _LOG_PATH_WINDOWS
+        if status_log_path:
+            return log_path + "\\status"
+        else:
+            return log_path
+    else:
+        return _LOG_PATH_LINUX
+
+
+def _env_var_or_default(var_name, default_val=""):
+    try:
+        return environ[var_name]
+    except KeyError:
+        return default_val
+
+
+def __get_customer_ikey():
```

Comment:
Im sure we do iKey parsing elsewhere.  Is there a way to centralize this logic?

Reply:
I believe this logic is available in the [exporter](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_connection_string_parser.py#L22)

Reply:
Cool. I'll switch to ConnectionStringParser().instrumentation_key

Reply:
done